app/api/api_v1/routers/lookups/geo_stats.py

```diff
@@ -22,8 +22,8 @@ class GeoStatsResponse(BaseModel):
     federal: bool
     federal_details: str
     political_groups: str
-    global_emissions_percent: Optional[float] = None
-    climate_risk_index: Optional[float] = None
+    global_emissions_percent: Optional[str] = None
+    climate_risk_index: Optional[str] = None
```

Comment:
I woud have thought we'd want to stick with floats here, how come we're changing it?

Reply:
Ah fair enough, yeah I would have thought we'd want a null value from the backend that could be used by the frontend to indicate if its not available. But happy for that to be one for another day!